src/Parser/Rogue/Subtlety/Modules/Talents/DarkShadow.js

```diff
@@ -0,0 +1,23 @@
+import Analyzer from 'Parser/Core/Analyzer';
+import Enemies from 'Parser/Core/Modules/Enemies';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+import SPELLS from 'common/SPELLS';
+import ShadowDance from './../Features/ShadowDance';
+
+class DarkShadow extends Analyzer {
+  static dependencies = {
+    enemies: Enemies,
+    combatants: Combatants,
+    shadowDance: ShadowDance,
+  };
+
+  bonusDmg = 0;
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasTalent(SPELLS.DARK_SHADOW_TALENT.id);
+  }
+
+}
```

Comment:
I'm confused, this module doesn't appear to do anything?

Reply:
Its base for 2 other DarkShadow modules, right now only checks if they need to be enabled